packages/adapter-next/public/AlternateGrid/javascript.jsx

```diff
@@ -58,9 +58,9 @@ const PascalNameToReplace = ({ slice }) => {
 							</div>
 						)}
 					</div>
-					{slice.items.length > 0 && (
+					{slice.primary.items.length > 0 && (
```

Comment:
Is there a plan to eventually move away from `primary` and/or change it to something like `data` or `fields`?

Reply:
That's a good question, we discussed it a while back because we wanted to do it at some point.

Here is the conclusion: https://prismic-team.slack.com/archives/C02L3FN3AJK/p1712825788916059?thread_ts=1712751482.676019&cid=C02L3FN3AJK

Reply:
Thanks for the context.